system_tests/noxfile.py

```diff
@@ -167,66 +169,88 @@ def configure_cloud_sdk(session, application_default_credentials, project=False)
 
 # Test sesssions
 
+TEST_DEPENDENCIES = ["pytest", "requests"]
 
-def session_service_account(session):
-    session.virtualenv = False
+
+@nox.session(python=["2.7", "3.7"])
```

Comment:
`["2.7", "3.7"]` is pretty redundant throughout this file. Let's make a constant for the python versions.